src/Application.cc

```diff
@@ -56,8 +56,12 @@ namespace ignition
       /// these until it is ok to unload the plugin's shared library.
       public: std::vector<std::shared_ptr<Plugin>> pluginsAdded;
 
+      /// \brief Deprecated environment variable which holds paths to look for
```

Comment:
Should we add a `TODO`s as a reminder to remove in the future?

Reply:
I usually search for "deprecated" when I'm doing a round to remove deprecations, that's one reason why I spelled out the word "deprecated" here. I could add a TODO but it may be a bit redundant. We don't add TODOs for the `IGN_DEPRECATED` macros, for example 🤔

Reply:
Sounds good to me